pkg/server/server.go

```diff
@@ -811,6 +812,17 @@ func (s *Server) Start(ctx context.Context) error {
 	if err != nil {
 		return errors.Wrap(err, "inspecting engines")
 	}
+	if runtime.GOOS != "windows" {
+		timer := time.NewTimer(time.Second * 30)
+		go func() {
+			<-timer.C
+			log.Shout(context.Background(), log.Severity_WARNING,
+				"Server is timing out while waiting for other nodes in the cluster.\n\n"+
+					"Please restart the other nodes without waiting for this one to finish. \n"+
+					"One way to do this is by appending & to your command instead of using --background. \n")
```

Comment:
Assuming that the command is using `--background` seems likely to be confusing to the majority of people that aren't using it. If we aren't going to check whether it was set anymore, I'd personally much prefer to just leave this part out. The important bit is telling the user that the other nodes need to be restarted before the currently running command will succeed.

Sorry for the bike shed!

Reply:
while we're bike shedding -- we should be telling users to "start" the other nodes, not "restart" them? We don't want someone who is running all the nodes (but perhaps messed up the cluster connectivity) to restart nodes thinking that might solve the problem.

So the message could be:

```
The server appears to be unable to contact the other nodes in the cluster. Please try

- starting the other nodes, if you haven't already
- double-checking that the `--join` and `--host` flags are set up correctly
- avoid using the `--background` flag.

If problems persist, please see [link to doc page].
```

Reply:
On a related note, it just occurred to me that `func maybeRerunBackground()` might be a good place to print a friendly reminder that the `--background` flag may be inappropriate for many use cases.